python_modules/libraries/dagster-aws/dagster_aws_tests/ecs_tests/launcher_tests/test_launching.py

```diff
@@ -148,8 +147,9 @@ def test_launcher_dont_use_current_task(
     assert container_definition["image"] == image
     assert not container_definition.get("entryPoint")
     assert not container_definition.get("dependsOn")
-    # It takes in the environment configured on the instance
-    assert container_definition["environment"] == environment
```

Comment:
is this asserting against important behavior? All i did was append the new env var and it seems to change this inheritance scheme?